src/bulma/pages/files/components/Preview.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <modal class="preview-modal"
+        show
+        v-on="$listeners">
+        <div class="box has-background-light">
+            <div class="detail">
+                <strong class="name">{{file.name}}</strong>
+                <a class="button is-naked" :href="url" target="_blank">
+                    <span class="icon">
+                        <fa icon="external-link-alt"/>
+                    </span>
+                </a>
+            </div>
+            <iframe :src="url"/>
+        </div>
+    </modal>
+</template>
+
+<script>
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
+import { Modal } from '@enso-ui/bulma';
```

Comment:
in packages we want to import directly from deps, ot the aggregator (bulma).

we also need to make sure that the deps are added to package.json